packages/block-library/src/post-comments-form/edit.js

```diff
@@ -67,8 +67,59 @@ export default function PostCommentsFormEdit( {
 					</Warning>
 				) }
 
-				{ ( 'open' === commentStatus || isInSiteEditor ) &&
-					__( 'Post Comments Form' ) }
+				{ ( 'open' === commentStatus || isInSiteEditor ) && (
+					<div className="wp-block-post-comments">
```

Comment:
I think the class name here should be `wp-block-post-comments-form`

Reply:
you're right, thank you 🙂